models/evc.py

```diff
@@ -1367,10 +1367,13 @@ def handle_link_up(self, link):
             link(Link): Link affected by link.up event.
 
         """
-        if self.is_intra_switch():
-            return True
-
-        if self.is_using_primary_path():
+        if any(
+            (
+                self.is_enabled() and self.is_active(),
```

Comment:
In terms of code cov looks good, if you could explicitly also cover the new added conditional on [`test_handle_link_up`](https://github.com/kytos-ng/mef_eline/blob/master/tests/unit/test_main.py#L1798) that'd be great. The associated unit test is already testing multiple cases, but if someone were to remove this conditional here unit tests wouldn't fail, so ideally, let's also enforce at unit test level to ensure the new logical conditional is covered and will fail if removed.

Reply:
Added in a test for these. Additionally, fixed what I think was a defect. Handle_link_up would always return true, when it seems that the intention was that it should return false if the evc can't be deployed.